packages/web/src/features/chat/utils.ts

```
@@ -0,0 +1,306 @@
+                    unit: 'character',
+                })
+                : Editor.before(editor, point, { unit: 'character' })
+
+        const wordNext =
+            next &&
+            Editor.string(
+                editor,
+                direction === 'right' ? { anchor: point, focus: next } : { anchor: next, focus: point },
+            )
+
+        const last = wordNext && wordNext[direction === 'right' ? 0 : wordNext.length - 1]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)